src/CommunityToolkit.Maui.MediaView.Analyzers/CommunityToolkit.Maui.MediaView.Analyzers.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
I don't quite see how this ships to customers. Is there a missing packaging project?

The same applies to the existing `CommunityToolkit.Maui.Analyzers.csproj`

Reply:
The assemblies are built in the pipeline and referenced as binaries and packed up through the other NuGet packable projects.

Reply:
@jfversluis Could you elaborate more if you don't mind?

All what I see in the CI pipeline is `dotnet build -c Release`, when/how exactly they are packed?

Reply:
Oh I see now.

```xml
    <None Include="..\CommunityToolkit.Maui.MediaView.Analyzers\bin\$(Configuration)\netstandard2.0\CommunityToolkit.Maui.MediaView.Analyzers.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
    <None Include="..\CommunityToolkit.Maui.MediaView.Analyzers.CodeFixes\bin\$(Configuration)\netstandard2.0\CommunityToolkit.Maui.MediaView.Analyzers.CodeFixes.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```